src/extension/rn-extension.ts

```diff
@@ -1,6 +1,17 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for details.
 
+import * as fs from "fs";
+
+try {
+    fs.statSync(`${__filename}.map`); // We check if source maps are available
```

Comment:
Do we actually need to check for the file? Won't line 9 fail if the .map file is not found?